src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -581,7 +678,7 @@ function App() {
           getUser();
           break;
         case "signInWithRedirect_failure":
-          setError("An error has ocurred during the oauth flow.");
+          setError("An error has ocurred during the Oauth flow.");
```

Comment:
nit: the quotes are inconsistent through out the same example 😅

Reply:
+1

Reply:
updated